.github/workflows/ci.yml

```diff
@@ -23,9 +23,10 @@ jobs:
       TOX_COMMAND: "tox --skip-missing-interpreters false"
       RESOLWE_POSTGRESQL_PORT: 55433
       RESOLWE_REDIS_PORT: 56380
+      RESOLWE_TEST_ONLY_CHANGES_TO: "origin/master"
     strategy:
       matrix:
-        toxenv: ["py311", "py312"]
+        toxenv: ["py311-partial", "py312-partial"]
```

Comment:
Why do we run tests with two versions of Python? Most of the tests are for processes and thus don't make any difference, except that they take double the amount of time.

Reply:
It's not exactly double time, but certainly double resources.

Not all tests are processes though. Some (variant tests for instance) are viewset tests. 

We could remove the `Python 3.11` tests since we use `Python 3.12` in the production.